src/content/advanced/prometheus-remotewrite/index.md

```diff
@@ -66,3 +68,7 @@ spec:
       username: ...
     name: grafana-cloud
 ```
+
+## RemoteWrite CRD
+
+The RemoteWrite CRD documentation can be found [here](https://github.com/giantswarm/prometheus-meta-operator/blob/master/config/crd/monitoring.giantswarm.io_remotewrites.yaml) or on any management cluster using `kubectl explain`.
```

Comment:
I believe, it's better to generate an API reference for the CRD:
like we have here: https://docs.giantswarm.io/ui-api/management-api/crd/

Reply:
Or otherwise, we may share a link like this: https://doc.crds.dev/github.com/giantswarm/prometheus-meta-operator/monitoring.giantswarm.io/RemoteWrite/v1alpha1@v4.5.1

Although, [doc.crds.dev] is not a giant swarm tool. it was created by upbound.

Reply:
Can you link the docs.crds.dev as a suggestion ?

Reply:
Yes!

Reply:
And I think we can bring this topic to SIG DEV, so we can create our own docs.crds, as it's an opensource tool

Reply:
You can bring this to sig-dev for sure :)